content/academy/apify_platform/getting_started/apify_client.md

```diff
@@ -58,7 +58,7 @@ client = ApifyClient(token='YOUR_TOKEN')
 
 > If you are planning on publishing your code to a public Github/Gitlab repository or anywhere else online, be sure to set your API token as en environment variable, and never hardcode it directly into your script.
 
-Now that we've got our instance, we can point to an actor using the `client.actor()` function, then call the actor with some input with the `.call()` function.
+Now that we've got our instance, we can point to an actor using the `client.actor()` function, then call the actor with some input with the `.call()` function - the first parameter of which is the input for the actor.
```

Comment:
```suggestion
Now that we've got our instance, we can point to an actor using the [`client.actor()`](https://docs.apify.com/apify-client-js#apifyclient-actor) function, then call the actor with some input with the [`.call()`](https://docs.apify.com/apify-client-js#actorclient-call) function - the first parameter of which is the input for the actor.
```

Reply:
I'll add docs links for these types of methods from now on.